src/components/ReportActionItem/ReportPreview.js

```diff
@@ -153,7 +162,7 @@ function ReportPreview(props) {
     const numberOfScanningReceipts = _.filter(transactionsWithReceipts, (transaction) => TransactionUtils.isReceiptBeingScanned(transaction)).length;
     const hasReceipts = transactionsWithReceipts.length > 0;
     const isScanning = hasReceipts && areAllRequestsBeingSmartScanned;
-    const hasErrors = hasReceipts && hasMissingSmartscanFields;
+    const hasErrors = hasReceipts && hasMissingSmartscanFields || (canUseViolations && ReportUtils.hasViolations(props.iouReportID, props.transactionViolations));
```

Comment:
This should fix the lint check
```suggestion
    const hasErrors = (hasReceipts && hasMissingSmartscanFields) || (canUseViolations && ReportUtils.hasViolations(props.iouReportID, props.transactionViolations));
```